scripts/plots/wafs/exevs_wafs_atmos_plots.sh

```diff
@@ -1,121 +1,56 @@
-    mkdir -p $OUTPUT_BASE_DIR
-
-    source $HOMEevs/parm/evs_config/wafs/config.evs.wafs.standalone
-
-    if [ $OBSERVATION = "GCIP" ] ; then
-        stat_file_suffix=`echo $VAR1_NAME | tr '[:upper:]' '[:lower:]'`
-    elif [ $OBSERVATION = "GFS" ] ; then
-	stat_file_suffix="uvt$resolution"
-	#===================================================================================================#
-	#========== Turn off Wind Direction verification until its RMSE gets supported by METplus ==========#
-	# stat_file_suffix="'uvt'$resolution 'wdir'$resolution"
-	#===================================================================================================#
-    fi
-
-    cd $DATA    
-
-    for suffix in $stat_file_suffix ; do
-	if [[ $suffix =~ 'uvt' ]] ; then
-	    export LINE_TYPE="SL1L2"
-	elif [[ $suffix =~ 'wdir' ]] ; then
-	    export LINE_TYPE="VL1L2"
-	else
-	    export LINE_TYPE="ctc"
+    for ndays in $DAYS_LIST ; do
+	if [ $ndays -le 90 ] ; then
+	    loopFHOURS="all"
 	fi
-
-	# Re-organize data for plotting
-	n=0
-	while [[ $n -le $NDAYS ]] ; do
-	    day=`date -d "$VDATE - $n days" +%Y%m%d`
-	    if [ $NDAYS -gt 90 ] ; then
-		yyyy=`echo $day | cut -c 1-4`
-		sourcefile=$COMIN/$yyyy/$NET.stats.$MODELNAME.$RUN.${VERIF_CASE}_${stat_file_suffix}.v$day.stat
-	    else
-		sourcefile=$EVSINstat/${MODELNAME}.$day/$NET.stats.$MODELNAME.$RUN.${VERIF_CASE}_${stat_file_suffix}.v$day.stat
-	    fi
-	    targetfile=$OUTPUT_BASE_DIR/$NET.stats.$MODELNAME.$RUN.${VERIF_CASE}_${stat_file_suffix}.v$day.stat
-	    if [[ ! -f "$targetfile" ]] ; then
-		if [[ -f "$sourcefile" ]] ; then
-		    ln -s $sourcefile $OUTPUT_BASE_DIR/.
-		fi
-	    fi
-	    n=$((n+1))
-	done
-
-	for PLOT_TYPE in $plot_types ; do
-	    export PLOT_TYPE
-	    if [[ $PLOT_TYPE = "roc_curve" ]] ; then
-		export STATS="farate,pod"
-	    elif [[ $PLOT_TYPE = "fbias" ]] ; then
-		export STATS="fbias"
-	    elif [[ $PLOT_TYPE = "time_series" ]] ; then
-		export STATS="rmse"
-	    fi
-	    # Set the config and run python scripts to generate plots
-	    $HOMEevs/parm/evs_config/wafs/config.evs.plots.wafs.atmos
-	    export err=$?; err_chk
+	for subregion in $VX_MASK_ALL ; do
+	    for hh in $loopFHOURS ; do
+		for plot_type in $plot_types ; do
+		    if [ `echo $MPIRUN | cut -d " " -f1` = 'srun' ] ; then
+			echo $ic $USHevs/evs_wafs_atmos_plots.sh $observation $ndays $subregion $hh $plot_type >> wafs_plots.cmdfile
+		    else
+			echo $USHevs/evs_wafs_atmos_plots.sh $observation $ndays $subregion $hh $plot_type >> wafs_plots.cmdfile
+			export MP_PGMMODEL=mpmd
+		    fi
+		    ic=$(( ic + 1 ))
+		done
+	    done
 	done
     done
+done
+export MPIRUN="$MPIRUN -np $ic -cpu-bind verbose,core cfp"
+$MPIRUN wafs_plots.cmdfile
 
+export err=$?; err_chk
+
+cd $DATAplot
+for ndays in $DAYS_LIST ; do
+    eval_period="last${ndays}days"
+    tar -cvf $COMOUT/$NET.$STEP.${COMPONENT}.${RUN}.${VERIF_CASE}.$eval_period.v${VDATE}.tar *${eval_period}*png
 done
 
```

Comment:
@YaliMao-NOAA For plotting, we need to tar up the plots in the working directory, and then wrap this around a SENDCOM block.  It kind of looks like this:

```
if [ $SENDCOM = "YES" ]; then
 if [ -s evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar ]; then
  cp -v evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar $COMOUTplots
 fi
fi
```

I don't think you can tar directly to COMOUT in this manner, especially since SENDCOM is required.  

Also you will need to add dbn alerts.  To do this, you'd need to set SENDDBN=YES in your plots J-job, and then have a block that looks like this:

```
if [ $SENDDBN = YES ] ; then     
 $DBNROOT/bin/dbn_alert MODEL EVS_RZDM $job $COMOUTplots/evs.plots.${COMPONENT}.${RUN}.${VERIF_CASE}.last31days.v${VDATE}.tar
fi
```

The words MODEL and EVS_RZDM do not need to be changed, but you can change the tar file name in this command.  This is how the plot tarfile gets sent to the RZDM server for posting on the web.  We'll need to make these changes as well.

Reply:
@PerryShafran-NOAA Thank you for the great suggestions. I am adding them to my scripts.

Reply:
@PerryShafran-NOAA Do I need to add dbn_alerts since WAFS plottting is not going to be sent to RZDM by NCO?

Reply:
Oh, I don't know.  

@AliciaBentley-NOAA do you know the answer to Yali's question?

Reply:
I can add dbn_alerts anyway. I have a standalone script which transfers the tar files to RZDM, untar to the target folder on RZDM then delete the tar files right away.

Reply:
@PerryShafran-NOAA @YaliMao-NOAA I believe that these scripts plot several years of statistics on one plot. Is that correct? If this is the case, then these scripts will always be run in dev and will not issue dbnet alerts in operations.  

I will also note that WAFS **31-day and 90-day** plotting scripts are intended to become operational in EVS v2.0. Please see the [Fixes and Additions in EVS v2.0](https://docs.google.com/document/d/1vOGnCwCM3COUhQlTqHyBZrYSCPx9b6AYnxLc5NtVGEk/edit?pli=1#heading=h.ebjtpqopgwje) document and refer back to our group emails from May 2023 called "WAFS code updates for EVSv1" for more information about this. Thanks! 

CC @ShelleyMelchior-NOAA

Reply:
@AliciaBentley-NOAA I believe that this PR is for the 31/90 day plots.  Are these plots meant to be sent to rzdm?

Reply:
@PerryShafran-NOAA Yes, the 30-day and 91-day WAFS plots will be run in operations in EVS v2.0, so the code needs to meet EE2 standards and needs to include dbnet alerts so that NCO can send plots to emcrzdm.

Reply:
@AliciaBentley-NOAA Thank you for the clarification. 
@PerryShafran-NOAA I've added both SENDCOM and SENDDBN as you suggested. Thank you!